backend/chess/pieces.py

```diff
@@ -35,7 +33,21 @@ def get_piece_name(piece: int) -> str:
         ROOK: "rook",
         QUEEN: "queen",
         KING: "king",
-        WINGED_KNIGHT: "winged_knight",
+        WINGED_KNIGHT: "wingedknight",
         HOPLITE: "hoplite",
         WARELEFANT: "warelefant",
     }[piece]
+
+
+def get_upgraded_variant(piece: int) -> int:
+    return {
+        PAWN: HOPLITE,
+        BISHOP: BISHOP,

```

Comment:
alles wat niet geupgrade kan wordne is onnodig toch?

Reply:
Nope, gaat via n index dus dan doet ie het nie en zo handeld hij ook meteen de niet upgradebare dingen

Reply:
ok is geod